File: source/examples/Query/Comparison.cs
Debug.WriteLine("High priority tasks: "
        + tasks.Where(t => t.Priority > 5));

Debug.WriteLine("Just started or short tasks: "
    + tasks.Where(t => 1 <= t.ProgressMinutes && t.ProgressMinutes < 15));

Debug.WriteLine("Unassigned tasks: "
    + tasks.Where(t => t.Assignee == null));

Debug.WriteLine("Ali or Jamie's tasks: "
    + tasks.Where(t => new List<string> { "Ali", "Jamie" }.Contains(t.Assignee)));

File: source/examples/Query/Comparison.java
Log.i(TAG, "High priority tasks: " + tasksQuery.greaterThan("priority", 5).count());
Log.i(TAG, "Just-started or short tasks: " + tasksQuery.between("progressMinutes", 1, 10).count());
Log.i(TAG, "Unassigned tasks: " + tasksQuery.isNull("assignee").count());
Log.i(TAG, "Ali or Jamie's tasks: " + tasksQuery.in("assignee", new String[]{"Ali", "Jamie"}).count());
